app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+require 'bcrypt'
+
+module Chitter
+  module Models
+    class User
+
+      attr_reader :password
+      attr_accessor :password_confirmation
+
+      include DataMapper::Resource
+
+      validates_confirmation_of :password
+      validates_uniqueness_of :email
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.